contracts/finality/src/state/config.rs

```diff
@@ -18,6 +18,10 @@ pub fn get_config(deps: Deps<BabylonQuery>) -> StdResult<Config> {
     CONFIG.load(deps.storage)
 }
 
```

Comment:
Should `ADMIN` and `CONFIG` become private now?

Reply:
On second thought, this is kind of overkill. Those public containers are used everywhere in test code, and the improvement of using the accessor methods is minimal, both in terms of functionality and readability.